demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -439,12 +476,59 @@ def test_has_skipped_integration():
 
             return False
 
+        def marketplaces_match_server_type():
+            """
+            Checks if the test has a marketplace value, and if so- if it suits the server machine we are on.
+            """
+            test_server_types: Set[str] = set()
+            for marketplace in self.configuration.marketplaces or []:
+                test_server_types.update(
+                    MARKETPLACE_VERSIONS_TO_SERVER_TYPE[marketplace]
+                )
+            self.log_info(
+                f"MARKETPLACES FOR TEST PLAYBOOK:\n"
+                f"{self.configuration.marketplaces=} ({test_server_types=})\n"
+                f"for {self.configuration.playbook_id=}\n"
+                f"with {self.configuration.test_instance_names=}\n"
+                f"in {self.build_context.server_type=}"
+            )
+            if not test_server_types:
+                return True
```

Comment:
please explain with just a few words, on each `return`, why we return it - it's a bit unclear